### PR TITLE
Feature 2558: bandit (pds-multi) system test

### LINKED ISSUE #2558: System Test for Bandit (PDS-Multi)
### Situation
The Bandit Sarif report has currently been extended with the Common Weakness Enumeration (CWE) taxonomy, see #2556. As a test engineer I would like to ensure that this fuctionality works as intended.

### Solution
Run a system test for Bandit (PDS-Multi) analogous to GoSec ( https://github.com/mercedes-benz/sechub/tree/develop/sechub-pds-solutions/gosec/tests )

### PR DESCRIPTION
**Pull request by alexdd**
partly solves #2558 

This branch is based on https://github.com/mercedes-benz/sechub/pull/2557

### CODE REVIEW

File: sechub-pds-solutions/multi/tests/systemtest_local.json [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+    } ],
+    "execute" : {
+      "runSecHubJob" : {
+        "uploads" : [ {
+          "sourceFolder" : "."
+        } ],
+        "codeScan" : { }
+      }
+    },
+    "assert" : [ {
```
Comment (Jeeppler) [inline sechub-pds-solutions/multi/tests/systemtest_local.json:68]: Please add an ` assert -> equalsFile` statement. For more information have a look at the systemtest in the `pds-base` solution. Keep in mind you have to ensure put a placeholder for the SecHub Job UUID.

**Review by Jeeppler — CHANGES_REQUESTED** [review]
See comments.

**Review by Jeeppler — CHANGES_REQUESTED** [review]
See comments.

**Comment by CLA-Mercedes-Benz** [comment]
[CLA assistant check](https://cla.opensource.mercedes-benz.com/mercedes-benz/sechub?pullRequest=2576) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.opensource.mercedes-benz.com/mercedes-benz/sechub?pullRequest=2576) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla.opensource.mercedes-benz.com/check/mercedes-benz/sechub?pullRequest=2576) it.</sub>

**Review by Jeeppler — APPROVED** [review]
Looks good.

**Pull request merged**